app/models/replication.rb

```diff
@@ -0,0 +1,4 @@
+class Replication < ActiveRecord::Base #ResourceUsage why doesn't deriving work?
```

Comment:
use single table inheritance:
http://samurails.com/tutorial/single-table-inheritance-with-rails-4-part-1/
you'll need a string column called "type"